Sources/Puree/LogStore/FileLogStore.swift

```diff
@@ -56,8 +56,8 @@ public class FileLogStore: LogStore {
 
     public static let `default` = FileLogStore()
 
-    func fileURL(for outputTagPattern: String) -> URL {
-        let fileName = "\(FileLogStore.baseFileName)_\(outputTagPattern)"
+    private func fileURL(for group: String) -> URL {
+        let fileName = "\(FileLogStore.baseFileName)_\(group)"
         return baseDirectoryURL.appendingPathComponent(fileName)
```

Comment:
Not a problem in this PR, but it seems you have directory names that contain `*`!?